GNUmakefile

```diff
@@ -32,4 +32,4 @@ test: fmtcheck
 # Run acceptance tests
 .PHONY: testacc
 testacc: fmtcheck
-	TF_ACC=1 go test -count=1 $(TEST) -v $(TESTARGS) -timeout 120m
\ No newline at end of file
+	TF_ACC=1 TF_SCHEMA_PANIC_ON_ERROR=1 go test -count=1 $(TEST) -v $(TESTARGS) -timeout 120m -ldflags="-X=github.com/hashicorp/terraform-provider-googleworkspace/version.ProviderVersion=acc"
```

Comment:
panic on error is on by default in SDKv2

Reply:
ooooh, good to know! thanks!

Reply:
I don't even think it can be turned off anymore lol..